index.html

```diff
@@ -111,6 +113,13 @@ <h2>Community Blob Servers</h2>
       </p>
     </section>
 
+    <section>
+      <h2>Recently Changed Blobs</h2>
+      <div class="emoji-list" id="recently-changed-emoji">
+        <!-- react will be injected here -->
```

Comment:
Maybe put some loading text here?

Reply:
how about the whole section be hidden until loaded?

Reply:
I think it really depends on how fast the section loads.